api/api.go

```diff
@@ -75,6 +75,17 @@ func (api *API) enablePublicEndpoints() {
 	api.Router.Get("/filter-outputs/{id}", api.getFilterOutput)
 
 	api.Router.Get("/datasets/{dataset_id}/editions/{edition}/versions/{version}/json", api.getDatasetJSONHandler)
+
+	if api.cfg.EnableFilterOutputs {
+		permissions := middleware.NewPermissions(api.cfg.ZebedeeURL, api.cfg.EnablePermissionsAuth)
+		checkIdentity := dphandlers.IdentityWithHTTPClient(api.identityClient)
+
+		api.Router.
+			With(checkIdentity).
+			With(middleware.LogIdentity()).
+			With(permissions.Require(auth.Permissions{Read: true})).
```

Comment:
Should this be update auth.Permissions?

Reply:
I think its the name that adds the confusion; permissions is local object created with middleware.NewPermissions and different to auth.Permissions